src/highlight.js

```diff
@@ -53,7 +53,8 @@ https://highlightjs.org/
     classPrefix: 'hljs-',
     tabReplace: null,
     useBR: false,
-    languages: undefined
+    languages: undefined,
+    langAttribute: false
```

Comment:
`addLanguageAttr`

And could we add docs also `api.rst` would need to be updated to mention this.

Reply:
Changed. Better, more explicit.

Reply:
Now on docs also :)